population/redfish_list_example

```diff
@@ -0,0 +1,2 @@
+IPMI_IP,IPMI_USERNAME,IPMI_PASSWORD,PUBLIC_IP,
```

Comment:
remove comma from end of line 1 and/or maybe add `LAB` to end of each line? So it would look like `IPMI_IP,IPMI_USERNAME,IPMI_PASSWORD,PUBLIC_IP,LAB`